packages/identity-service/src/notifications/emails/otp.js

```diff
@@ -0,0 +1,361 @@
+    <style>@media only screen and (max-width:480px){u+.body .full-wrap{width:100%!important;width:100vw!important}}</style> 
+    <style>@-ms-viewport{width:device-width}</style> 
+    <!--[if (gte mso 9)|(IE)]>
+    <style type="text/css">table {border-collapse: collapse !important;}.outf14{font-size:14px !important;}    .w520px{ width: 520px !important;} .w520px{ width: 520px !important;} .not_for_outlook{mso-hide: all !important;display: none !important;font-size:0;max-height:0;line-height: 0;mso-hide: all;}.outpadding{padding-left: 0 !important;padding-right: 0 !important;}</style>
+    <![endif]--> 
+    </head> 
+    <body class="body" style="padding:0;margin:0"> 
+    <div class="full-wrap"> 
+    <table width="100%" border="0" cellspacing="0" cellpadding="0" class="full-wrap">
+    <tr><td align="center" bgcolor="#f5f5f5" style="line-height: normal; hyphens: none;">
+    <div>
+    <!--[if !mso]>
+    <!-->
+    <div class="mail_preheader" style="font-size: 0px; color: transparent; opacity: 0;">
+      <span style="font-family: Arial, Helvetica, sans-serif; font-size: 0px; color: transparent; line-height: 0px;"></span>
+    </div>
+    <!--
+    <![endif]-->
+  </div>  
+  <div>
+    <!--[if (gte mso 9)|(IE)]>
+    <table width="700" border="0" cellspacing="0" cellpadding="0" style="width: 700px;">
+    <tr><td>
+    <![endif]-->
+    <table border="0" cellspacing="0" cellpadding="0" width="100%" style="max-width: 700px;">
+    <tr><td align="center" valign="top" bgcolor="#f3f0f7" style="padding: 40px 10px;">
+      <div>
+        <!--[if (gte mso 9)|(IE)]>
+        <table width="250" border="0" cellspacing="0" cellpadding="0" style="width: 250px;">
+        <tr><td>
+        <![endif]-->
+        <table border="0" cellspacing="0" cellpadding="0" width="100%" style="max-width: 250px;">
+        <tr><td align="left" valign="top" style="padding: 4px;">
+          <img src="https://download.audius.co/welcome-email/i1867840199.png" width="240" height="64" alt="" border="0" style="display: block;">
+        </td></tr>
+        </table>
+        <!--[if (gte mso 9)|(IE)]>
+        </td></tr>
+        </table>
+        <![endif]-->
+      </div> 
+      <div style="height: 40px; line-height: 40px; font-size: 38px;">&nbsp;</div>
+      <div>
+        <!--[if (gte mso 9)|(IE)]>
+        <table width="600" border="0" cellspacing="0" cellpadding="0" style="width: 600px;">
+        <tr><td>
+        <![endif]-->
+        <table border="0" cellspacing="0" cellpadding="0" width="100%" style="max-width: 600px;">
+        <tr><td align="left" valign="top" bgcolor="#ffffff" style="border-radius: 12px; box-shadow: rgba(0, 0, 0, 0.07) 0px 8px 16px;">
```

Comment:
Curious how you generated this html, based off our other email templates?

Reply:
yep